### PR TITLE
refactor: remove superfluous changes for card swiping

### PR DESCRIPTION
**Pull request by csiyang**
# Description

copilot:summary

- Link to Basecamp Todo

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

copilot:walkthrough

### CODE REVIEW

File: apps/journeys/src/components/Conductor/JourneyRenderer/JourneyRenderer.tsx
Comment (tataihono) [inline apps/journeys/src/components/Conductor/JourneyRenderer/JourneyRenderer.tsx:1]: change name to DynamicCardList. The full journey isn't really rendered here (footer, header etc).

File: apps/journeys/src/components/EmbeddedPreview/EmbeddedPreview.spec.tsx
Comment (tataihono) [inline apps/journeys/src/components/EmbeddedPreview/EmbeddedPreview.spec.tsx:1]: revert

File: libs/journeys/ui/src/components/BlockRenderer/BlockRenderer.tsx
Comment (tataihono) [inline libs/journeys/ui/src/components/BlockRenderer/BlockRenderer.tsx:1]: revert

File: libs/journeys/ui/src/components/StepHeader/PaginationBullets/Bullet/Bullet.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import CircleIcon from '@mui/icons-material/Circle'
+import { ReactElement } from 'react'
+
+type BulletVariant = 'active' | 'adjacent' | 'default'
+
+interface BulletProps {
+  variant: BulletVariant
+  // left handles the spacing and animation of bullets
```
Comment (tataihono) [inline libs/journeys/ui/src/components/StepHeader/PaginationBullets/Bullet/Bullet.tsx:8]: jsdoc.
```suggestion
  /** left handles the spacing and animation of bullets */
```

File: libs/locales/en/apps-journeys-admin.json
Comment (tataihono) [inline libs/locales/en/apps-journeys-admin.json:1]: revert

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/journeys/src/components/EmbeddedPreview/EmbeddedPreview.spec.tsx
Comment (csiyang) [reply apps/journeys/src/components/EmbeddedPreview/EmbeddedPreview.spec.tsx:1]: this one needs to stay to fix the test

**Review by tataihono — APPROVED** [review]

**Pull request merged**